engine/consensus/message_hub/message_hub.go

```diff
@@ -485,8 +506,9 @@ func (h *MessageHub) Process(channel channels.Channel, originID flow.Identifier,
 				Hex("origin_id", originID[:]).
 				Hex("block_id", msg.BlockID[:]).
 				Uint64("view", msg.View).
+				Hex("sig_data", msg.SigData[:]).
```

Comment:
```suggestion
```
We should be careful when logging slices especially from potentially byzantine nodes. Not very useful to log it but kinda dangerous.